amazon/ion/util.py

```diff
@@ -217,15 +226,106 @@ def bit_length(value):
             return 0
         return len(bin(abs(value))) - 2
 
+
     def total_seconds(td):
         return (td.microseconds + (td.seconds + td.days * 24 * 3600) * 10 ** 6) / 10 ** 6
 else:
     def bit_length(value):
         return value.bit_length()
 
+
     def total_seconds(td):
         return td.total_seconds()
 
-
 bit_length.__doc__ = 'Returns the bit length of an integer'
 total_seconds.__doc__ = 'Timedelta ``total_seconds`` with backported support in Python 2.6'
+
+
+class _EnumMetaClass(type):
+    """Metaclass for simple enumerations.
```

Comment:
Let's make it obvious in the doc string too.
```suggestion
    """This is a deprecated, internal-only class in ion-python—do NOT use it for any reason.

    Metaclass for simple enumerations.
```

Reply:
Addressed.